### ISSUE TITLE
Why Puffin-D can't be reproduced by using train_puffin_D.py

### ISSUE THREAD

**Issue by impfgao** [jzhoulab/puffin#4]
Excuse me, recently I want to reproduced puffin-D by myself, but after training for four days with batch size 10 using the train_puffin_D.py provided in GitHub along with the associated source data, the average correlation coefficient across five organizations is only around 35%. This doesn't seem to be an issue of insufficient training time. In fact, the model's correlation coefficient on the test set has already stabilized. Could it be due to incomplete training code or data, or perhaps some minor errors exist? Please kindly provide possible reasons. Thank you immensely.!

**Comment by KseniiaDundyk** [comment]
It's possible that the issue you're experiencing could be due to the lower batch size, because batch normalization tends to be sensitive to batch size variations, which can affect its performance. To solve this problem  you can try reducing the learning rate.
Hope it helps!

**Comment by impfgao** [comment]
Thanks for your answer! The problem has been solved, it is indeed a matter of batch size.
